webdriver/support/fixtures.py

```diff
@@ -142,3 +144,41 @@ def inner(path, query="", fragment=""):
                                   fragment))
         return rv
     return inner
+
+def create_dialog(session):
+    """Create a dialog (one of "alert", "prompt", or "confirm") and provide a
+    function to validate that the dialog has been "handled" (either accepted or
+    dismissed) by returning some value."""
+
+    def create_dialog(dialog_type, text=None, result_var=None):
```

Comment:
Although I do like cheese a lot (it's Havarti for me these days), I'm not so
sure about that API. It seems open to the same criticism that you had for my
original `create_dialog` implementation--the returned value is something (a
global identifier that will hold the result of the expression after dismissal)
very different than what you would expect (a representation of the dialog).
The current implementation is more verbose than both of our ideas, but it's
also the most direct about what is going on.